packages/strategy-tests/src/lib.rs

```diff
@@ -1328,10 +1330,25 @@ impl Strategy {
         if block_info.height == config.start_block_height
             && self.start_identities.number_of_identities > 0
         {
+            // Make sure we insert a key for transfers
+            let mut key_maps: KeyMaps = [(
+                Purpose::TRANSFER,
+                [(SecurityLevel::CRITICAL, vec![KeyType::ECDSA_SECP256K1])].into(),
+            )]
+            .into();
+            let mut new_key_count = self.start_identities.keys_per_identity - 1;
+
+            // Also an authentication key with critical security level for documents
+            key_maps.insert(
+                Purpose::AUTHENTICATION,
+                [(SecurityLevel::CRITICAL, vec![KeyType::ECDSA_SECP256K1])].into(),
+            );
+            new_key_count -= 1;
```

Comment:
I on purpose didn't want to add the transfer key to all identities.

Reply:
Fixed. I added extra_keys field for StartIdentities struct and the users can select whether or not to add a transfer key in the TUI